libsolidity/parsing/ParserBase.h

```diff
@@ -23,6 +23,7 @@
 #pragma once
 
 #include <memory>
+#include <libsolidity/interface/ErrorReporter.h>
 #include <libsolidity/interface/Exceptions.h>
```

Comment:
The ErrorReporter include?  ErrorReporter is used in the file, and at least in my HEAD, there is no `Exceptions.h` declare.